openslides_backend/models/fields.py

```diff
@@ -319,8 +318,10 @@ def get_schema(self) -> Schema:
         return self.extend_schema(super().get_schema(), **fqid_list_schema)
 
     def validate(self, value: Any, payload: Dict[str, Any] = {}) -> Any:
-        assert isinstance(value, list)
-        return [string_to_fqid(fqid) for fqid in value]
+        if value or self.required:
```

Comment:
I would suggest
```suggestion
        if value is not None or self.required:
```
to remove the implication that `0` or something similar could be a valid value and also remove the possibility that it becomes one by accident

Reply:
done